dev-libs/libusb-compat/libusb_compat-0.1.7.recipe

```diff
@@ -65,11 +63,12 @@ INSTALL()
 
 	rm $libDir/libusb*.la
 
-	# set up the develop directory correctly
-	prepareInstalledDevelLibs libusb-0.1 libusb
-	fixPkgconfig libusb-0.1
-
```

Comment:
Keep this, at this moment you will end up with a headers only develop package, move libusb-config also to the devel package, start using commandSuffix etc to the recipe:
```diff
-	# set up the develop directory correctly
-	prepareInstalledDevelLibs libusb-0.1 libusb
-	fixPkgconfig libusb-0.1
-
+	# set up the develop directory correctly
+	prepareInstalledDevelLibs libusb-0.1 libusb
+	fixPkgconfig
+
```

Reply:
Fixed